apps/web/pages/team/[slug]/[type].tsx

```diff
@@ -67,30 +76,33 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   if (!team) {
     return {
       notFound: true,
-    };
+    } as const;
```

Comment:
It allows reusing team/[slug]/[type] in team/[slug]/[type]/embed without TS errors.